finalPrep/modZeroHero.js

```diff
@@ -74,11 +142,46 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 
 // - Create 2 instances of your SuperHero class
 
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+  }
+    static archNemesis = "The Syntax Error"
```

Comment:
Hm, this may work- I'm really not sure! But you can assign these much like your other attributes, like `this.archNemesis =` then you just assign a set value rather than a parameter of the constructor.